gunicorn/_compat.py

```diff
@@ -110,7 +110,7 @@ def _wrap_error(exc, mapping, key):
     six.reraise(new_err_cls, new_err,
                 exc.__traceback__ if hasattr(exc, '__traceback__') else sys.exc_info()[2])
 
-if PY33:
+if PY3:
```

Comment:
This branch was added to backport selectors module from Python 3.4. We can now drop the whole if branch, `gunicorn/selectors.py`, `tests/test_selectors.py`.

Reply:
How should this block look?

https://github.com/benoitc/gunicorn/blob/9c8695b04f38398ae5ebf0aad37b69475d378fb4/gunicorn/workers/gthread.py#L39-L42

Reply:
That code is wrong. There is no `asyncio.selectors` submodule. There are bunch of `from x import *` imports in `asyncio/__init__.py` and I assume there is a `import selectors` in one of those submodules. I think we can replace that snippet with `import selectors` (it was added in Python 3.4)

Reply:
✅